From ac615730861d430aa38435ecb55031316062d6f6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Fri, 25 Aug 2017 12:02:02 +0300
Subject: [PATCH 2/2] crt: Include _strftime_l in msvcrt.def.in

This fixes linking of libc++, which uses this function. libc++
needs a msvcrt new enough to provide this function (present
since msvcr90 in the numbered msvcrt DLLs).
---
 mingw-w64-crt/lib-common/msvcrt.def.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mingw-w64-crt/lib-common/msvcrt.def.in b/mingw-w64-crt/lib-common/msvcrt.def.in
index c63edd2..35c685d 100644
--- a/mingw-w64-crt/lib-common/msvcrt.def.in
+++ b/mingw-w64-crt/lib-common/msvcrt.def.in
@@ -1481,6 +1481,7 @@ strcspn
 strerror
 ; strerror_s replaced by emu
 strftime
+_strftime_l
 strlen
 strncat
 strncat_s
-- 
2.7.4

